### PR TITLE
Executed observation page

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds a page for Executed Observations, similar to for telescopes, instruments, and allocations, for display and upload.

### CODE REVIEW

File: skyportal/tests/frontend/test_upload_observations.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        os.path.join(
+            os.path.dirname(os.path.dirname(__file__)),
+            'data',
+            filename,
+        ),
+    )
+
+    driver.wait_for_xpath(f'//*[contains(., "{filename}")]')
+    submit_button_xpath = '//button[contains(.,"Submit")]'
+    driver.click_xpath(submit_button_xpath, scroll_parent=True)
```
Comment (profjsb) [inline skyportal/tests/frontend/test_upload_observations.py:70]: Can you now test that those observations are indeed there/uploaded?

Comment (profjsb) [reply skyportal/tests/frontend/test_upload_observations.py:70]: Please also test behavior when you upload 
 1) malformed observations
 2) observations without airmass and seeing
 3) identical observations to what was loaded by someone else.

(I dont know what behavior you expect in 3 above, should be documented anyway).

**Review by profjsb — CHANGES_REQUESTED** [review]

File: skyportal/handlers/api/observation.py [hunk 2]
```diff
@@ -514,3 +519,95 @@ def delete(self, observation_id):
...
+                Instrument.id == instrument_id,
+            )
+            .first()
+        )
+        if instrument is None:
+            return self.error(message=f"Missing instrument with ID {instrument_id}")
+
+        observation_data = pd.read_table(StringIO(observation_data), sep=",").to_dict(
+            orient='list'
+        )
```
[the remark this answers is not in the document]
Comment (mcoughlin) [reply skyportal/handlers/api/observation.py]: done.

File: skyportal/tests/frontend/test_upload_observations.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/tests/frontend/test_upload_observations.py:106]: Added. The identical observations one I do not currently test for as that is a log message, as opposed to something returned to the frontend. (just with the baselayer log). I made a few changes to the doc to make it clear what is actually required.

**Comment by mcoughlin** [comment]
@profjsb At @guynir42's suggestion, I went through and ran py.test with --pdb enabled, which led to mostly seeing places where a scroll_parent or timeout was useful. It was interesting indeed to see that the answers would be different each time, running on my own laptop 3 or 4 times.

**Review by profjsb — APPROVED** [review]

**Pull request merged**